sirepo/pkcli/admin.py

```diff
@@ -24,6 +24,27 @@
 import shutil
 
 
+def add_jupyterhub_role():
+    """Adds jupyterhub role to all users"""
+    import sirepo.server
+
+    sirepo.server.init()
+    import sirepo.auth
+    import sirepo.auth_db
+    import sirepo.pkcli.roles
+    import sirepo.template
+    import sqlalchemy.exc
+
+    sirepo.template.assert_sim_type('jupyterhublogin')
+    r = sirepo.auth.role_for_sim_type('jupyterhublogin')
```

Comment:
is the assert above necessary if role_for_sim_type is called?

Reply:
Yes, role_for_sim_type doesn't check if the sim_type is valid. It probably should. It is moot though because this code is removed 252b9c1d